tests/unit/test_utils.py

```diff
@@ -77,6 +78,17 @@ def list_for_dict():
     ]
 
 
+@pytest.fixture(scope='module')
+def dummy_strict_kwargs_cls():
+    """A sample class with @strict_kwargs constructor."""
+    class Dummy(object):
+        @strict_kwargs
+        def __init__(self, a, b=10, c=20, d=30, *args, **kwargs):
+            pass
```

Comment:
as said before (but below), you might want to store the values of the params passed.